src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -2,172 +2,193 @@
   name="stepTimeField.form"
   role="form"
   ng-submit="stepTimeField.createIndexPattern()"
-  class="kuiPanel"
 >
-  <div class="kuiPanelHeader">
-    <div class="kuiPanelHeaderSection">
-      <h2 class="kuiPanelHeader__title">
-        Step 2 of 2: Configure settings
-      </h2>
-    </div>
-  </div>
+  <h2 class="euiTitle">
+    Step 2 of 2: Configure settings
+  </h2>
+
+  <div class="euiSpacer euiSpacer--m"></div>
+
+  <p class="euiTextColor euiTextColor--subdued">
+    You've defined <strong>{{stepTimeField.indexPatternName}}</strong> as your index pattern. Now you can specify some settings before we create it.
+  </p>
+
+  <div class="euiSpacer euiSpacer--m"></div>
+
+  <div class="timeFieldNameLabel">
```

Comment:
We should use `euiFormRow` whenever we have groups of labels, inputs, and help text like this. Here's how we can use this pattern for lines 18-95:

```html
  <div class="euiFormRow">
    <div class="euiFormLabel">
      <div class="euiFlexGroup euiFlexGroup--justifyContentSpaceBetween">
        <div class="euiFlexItem euiFlexItem--flexGrowZero">
          <label
            id="timeFilterFieldSelectLabel"
            for="timeFilterFieldSelect"
          >
            Time Filter field name
          </label>
        </div>

        <div class="euiFlexItem euiFlexItem--flexGrowZero">
          <a
            ng-if="!stepTimeField.isFetchingTimeFieldOptions"
            class="euiLink euiText euiText--extraSmall"
            ng-click="stepTimeField.fetchTimeFieldOptions()"
            kbn-accessible-click
            aria-describedby="timeFilterFieldSelectLabel"
          >
            Refresh
          </a>

          <p
            ng-if="stepTimeField.isFetchingTimeFieldOptions"
            class="euiTextColor euiTextColor--subdued"
            aria-label="Please wait while we fetch your time field options"
            aria-describedby="timeFilterFieldSelectLabel"
          >
            <svg
              aria-hidden="true"
              class="euiIcon euiIcon--medium"
              xmlns="http://www.w3.org/2000/svg"
              width="16"
              height="16"
              viewBox="0 0 16 16"
            >
              <g fill="none" fill-rule="evenodd">
                <circle cx="5" cy="5" r="1" fill="#13252D" />
                <circle cx="10" cy="5" r="1" fill="#13252D" />
                <circle cx="7.5" cy="7.5" r="7" stroke="#13252D" />
                <path stroke="#13252D" stroke-linecap="round" stroke-linejoin="bevel"
                d="M3 8c1.167 2 2.667 3 4.5 3 1.833 0 3.333-1 4.5-3" />
              </g>
            </svg>
          </p>
        </div>
      </div>
    </div>

    <div class="euiFormControlLayout">
      <select
        ng-show="stepTimeField.canShowMainSelect()"
        id="timeFilterFieldSelect"
        aria-describedby="timeFilterFieldSelectHelpText1 timeFilterFieldSelectHelpText2"
        class="euiSelect euiSelect--large"
        data-test-subj="createIndexPatternTimeFieldSelect"
        ng-disabled="stepTimeField.isTimeFieldSelectDisabled()"
        ng-required="stepTimeField.hasTimeFieldOptions()"
        ng-options="option as option.display disable when option.isDisabled for option in stepTimeField.timeFieldOptions"
        ng-model="stepTimeField.selectedTimeFieldOption"
      ></select>

      <select
        ng-show="stepTimeField.canShowLoadingSelect()"
        id="timeFilterFieldSelect"
        aria-describedby="timeFilterFieldSelectHelpText1 timeFilterFieldSelectHelpText2"
        class="euiSelect euiSelect--large"
        data-test-subj="createIndexPatternTimeFieldSelect"
        disabled="disabled"
      >
        <option>Loading...</option>
      </select>

      <p
        ng-if="stepTimeField.canShowNoTimeBasedFieldsMessage()"
        class="euiTextColor euiTextColor--subdued euiText euiText--small"
      >
        The indices which match this index pattern don't contain any time fields.
      </p>
    </div>

    <div
      ng-if="stepTimeField.canShowHelpText()"
      class="euiFormHelpText euiFormRow__text"
      id="timeFilterFieldSelectHelpText1"
    >
      The Time Filter will use this field to filter your data by time.
    </div>

    <div
      ng-if="stepTimeField.canShowHelpText()"
      class="euiFormHelpText euiFormRow__text"
      id="timeFilterFieldSelectHelpText2"
    >
      You can choose not to have a time field, but you will not be able to narrow down your data by a time range.
    </div>
  </div>
```

This will render like this:

![image](https://user-images.githubusercontent.com/1238659/34229691-1176958e-e58a-11e7-8432-bf890e2f7304.png)